api/metrics/sidewalk_cafe.py

```diff
@@ -0,0 +1,43 @@
+from api.utils.database import rows_to_dicts
+
+class SidewalkCafeMetrics:
+    """
+    Metrics for sidewalk cafe permits data.
+    """
+
+    def __init__(self, con):
+        self.con = con
+    
+    def get_total_permits_day(self):
+        """
+        Returns the number of permits issued per day.
+        """
+        query = """
+        SELECT
+            issued_date_dt as date,
+            count(issued_date_dt) as number_of_permits
```

Comment:
Rename this column to `value` so that it automatically plugs into the timeline view.